examples/app_commands/slash_groups.py

```diff
@@ -5,7 +5,7 @@
 # If you use commands.Bot, @bot.slash_command should be used for
 # slash commands. You can use @bot.slash_command with discord.Bot as well
 
-math = bot.command_group(
+math = bot.create_group(
     "math", "Commands related to mathematics."

```

Comment:
Why is this changed?

Reply:
It was merged in from `master` as part of #576 and I'm just bad at keeping my commits clean for this branch